src/graph/GoExecutor.cpp

```diff
@@ -662,6 +660,47 @@ std::vector<VertexID> GoExecutor::getDstIdsFromResps(std::vector<RpcResponse>::i
     return std::vector<VertexID>(set.begin(), set.end());
 }
 
+std::vector<VertexID> GoExecutor::getDstIdsFromRespWithBackTrack(const RpcResponse &rpcResp) const {
+    // back trace in current step
+    // To avoid overlap in on step edges
+    // For example
+    // Dst , Src
+    // 6  ,  1
+    // 7  ,  6
+    // Will mistake lead to 7->6 if insert one by one
+    // So read all roots of current step first , then insert them
```

Comment:
Emm, i'd really appreciate that you could make these comments much more clear if you want the reader to understand what you want to fix.